### PR TITLE
update example github_search

### PR DESCRIPTION
**Pull request by hoangchungk53qx1**
Update Android targeting-s-version-31-and-above, update search Ui

### CODE REVIEW

**Comment by hoc081098** [comment]
Please remove all .ds_store file

File: example/flutter/github_search/lib/search_widget.dart [hunk 1]
```diff
@@ -49,40 +49,71 @@ class SearchScreenState extends State<SearchScreen> {
...
+    return Container(
+      height: 50,
+      child: Row(
+        crossAxisAlignment: CrossAxisAlignment.center,
+        children: [
+          const SizedBox(
+            width: 10,
+          ),
+          const Icon(
+            Icons.person_search,
```
Comment (hoc081098) [inline example/flutter/github_search/lib/search_widget.dart:86]:
```suggestion
            Icons.search,
```

File: example/flutter/github_search/lib/search_widget.dart [hunk 2]
```diff
@@ -49,40 +49,71 @@ class SearchScreenState extends State<SearchScreen> {
...
+            color: Colors.white,
+          ),
+          const SizedBox(
+            width: 10,
+          ),
+          Expanded(
+            child: TextFormField(
```
Comment (hoc081098) [inline example/flutter/github_search/lib/search_widget.dart:93]:
```suggestion
            child: TextField(
```

File: example/flutter/github_search/lib/search_widget.dart [hunk 3]
```diff
@@ -49,40 +49,71 @@ class SearchScreenState extends State<SearchScreen> {
...
+              textAlignVertical: TextAlignVertical.center,
+              textInputAction: TextInputAction.search,
+              style: const TextStyle(
+                fontSize: 14.0,
```
Comment (hoc081098) [inline example/flutter/github_search/lib/search_widget.dart:97]:
```suggestion
                fontSize: 18.0,
```

File: example/flutter/github_search/lib/search_widget.dart [hunk 4]
```diff
@@ -49,40 +49,71 @@ class SearchScreenState extends State<SearchScreen> {
...
             ],
           ),
         );
       },
     );
   }
 
+  Widget _buildSearchBar() {
+    return Container(
+      height: 50,
```
Comment (hoc081098) [inline example/flutter/github_search/lib/search_widget.dart:78]: Dont fix height 😂

**Review by hoc081098 — CHANGES_REQUESTED** [review]

File: example/flutter/github_search/lib/search_widget.dart [hunk 5]
```diff
@@ -49,40 +49,71 @@ class SearchScreenState extends State<SearchScreen> {
...
             ],
           ),
         );
       },
     );
   }
 
+  Widget _buildSearchBar() {
+    return Container(
+      padding: const EdgeInsets.symmetric(vertical: 10, horizontal: 10),
```
Comment (hoc081098) [inline example/flutter/github_search/lib/search_widget.dart:78]: Already applied padding when calling `_buildAppBar()`, we can adjust padding value at line 60 😃
```suggestion
```

**Review by hoc081098 — CHANGES_REQUESTED** [review]
Remove all .ds_store file

**Review by hoc081098 — CHANGES_REQUESTED** [review]
Should keep .metadata, remove .DS_Store file

**Review by hoc081098 — CHANGES_REQUESTED** [review]
`.metadata` should be added again 😂

**Comment by hoc081098** [comment]
Just revert ff9e960c28ce4ef13c4808ddd586a4ec0bfedc00

**Comment by AlphaAndOmage** [comment]
💯

File: example/flutter/github_search/lib/search_widget.dart (hunk 5 above)
Comment (hoangchungk53qx1) [reply example/flutter/github_search/lib/search_widget.dart]: padding content inside textfield search

**Review by hoc081098 — APPROVED** [review]

File: example/flutter/github_search/lib/search_widget.dart [hunk 6]
```diff
@@ -49,40 +49,71 @@ class SearchScreenState extends State<SearchScreen> {
...
             ],
           ),
         );
       },
     );
   }
 
+  Widget _buildSearchBar() {
+    return Container(
+      padding: const EdgeInsets.all(10),
```
Comment (hoc081098) [inline example/flutter/github_search/lib/search_widget.dart:78]:
```suggestion
      padding: const EdgeInsets.all(10),
```

**Pull request merged**